spec/actors/hyrax/actors/collections_membership_actor_spec.rb

```diff
@@ -100,6 +100,52 @@
       end
     end
 
+    context 'when passing `member_of_collection_ids`' do
+      let(:attributes) { { member_of_collection_ids: [collection.id], title: ['test'] } }
+
+      it 'adds it to the collection' do
+        skip 'this behavior past its deprecation sunset time and can be removed' if
+          Hyrax::VERSION.split('.').first.to_i >= 3
+
+        expect(subject.create(env)).to be true
+        expect(collection.reload.member_objects).to eq [curation_concern]
+      end
+
+      context "when work is in user's own collection" do
+        let(:attributes) { { member_of_collection_ids: [] } }
+
+        it "removes the work from that collection" do
+          skip 'this behavior past its deprecation sunset time and can be removed' if
+            Hyrax::VERSION.split('.').first.to_i >= 3
+
+          expect(subject.create(env)).to be true
+          expect(collection.reload.member_objects).to be_empty
+        end
+      end
+
+      describe "when work is in another user's collection" do
+        let(:other_user)    { create(:user) }
+        let(:collection)    { create(:collection, user: other_user, title: ['A good title']) }
+        let(:my_collection) { create(:collection, title: ['My good title']) }
+
+        let(:attributes) { { member_of_collection_ids: [my_collection.id] } }
+
+        before do
+          curation_concern.member_of_collections = [collection]
+          curation_concern.save!
+        end
+
+        it "doesn't remove the work from the other user's collection" do
+          skip 'this behavior past its deprecation sunset time and can be removed' if
+            Hyrax::VERSION.split('.').first.to_i >= 3
+
+          expect(subject.create(env)).to be true
+          expect(curation_concern.member_of_collections)
+            .to contain_exactly(collection, my_collection)
+        end
+      end
+    end
+
     context "updates env" do
```

Comment:
Need to also update the tests under `context "updated env"` to test that the update to env happens for both :member_of_collections_attributes and :member_of_collection_ids parameters.

Effected tests:  https://github.com/samvera/hyrax/blob/9c5fa41dd2e161ac70421aae573e0dea3c7a8e77/spec/actors/hyrax/actors/collections_membership_actor_spec.rb#L149-L203

Reply:
You can copy in the tests that use the old style parameter from here...
https://github.com/samvera/hyrax/blob/d67f598e992bcf5897c0731db1551ca8698cb04e/spec/actors/hyrax/actors/collections_membership_actor_spec.rb#L89-L129